### PR TITLE
Keep selected conversations in view with a warning when they no longer meet filters

### PR DESCRIPTION
**Pull request by marianamarasoiu**

### CODE REVIEW

File: webapp/lib/view.dart [hunk 1]
```diff
@@ -1137,6 +1147,10 @@ class ConversationSummary with LazyListViewItem {
     _checkboxHidden = !show;
     if (_selectCheckbox != null) _selectCheckbox.hidden = !show;
   }
+  void _showWarning(bool show) {
+    _warning = true;
```
Comment (danrubel) [inline webapp/lib/view.dart:1151]: `_warning = show;` ?

Comment (marianamarasoiu) [reply webapp/lib/view.dart]: Yes, sorry, fixed now, thanks for catching this, it would have been quite a pain to debug!

**Comment by lukechurch** [comment]
Thanks @marianamarasoiu - can we bring this in to start testing?

**Comment by lukechurch** [comment]
@marianamarasoiu - sorry this seems to have drifted, could you bring back up to head please. Ideally I'd like to start testing this over the next few days

**Pull request merged**